jupyterhub/templates/hub/deployment.yaml

```diff
@@ -1,30 +1,32 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
```

Comment:
We should make sure that upgrading from an older version of the chart continues to work after this change!

Reply:
- I wonder how Helm behaves, I figure the old deployment will be deleted fully and then recreated, but it should not be an issue I think.
- Would you suggest we simply test on some deployment as a way to make sure this works before merging?